packages/expect-puppeteer/src/utils.js

```diff
@@ -22,9 +22,8 @@ export const getContext = async (instance, pageFunction) => {
         handle: await instance.evaluateHandle(pageFunction),
       }
     case 'ElementHandle': {
-      const executionContext = await instance.executionContext()
       return {
-        page: await executionContext.frame(),
+        page: instance.frame,
```

Comment:
The way to access `ElementHandle` **frame** instance has changed as it was pointed [here](https://github.com/smooth-code/jest-puppeteer/issues/494#issuecomment-1233878303).